src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.CustomException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationValidator;
+    private static StorageDao storageDao;
+    private static User userIsValid;
+    private static User invalidUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationValidator = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        userIsValid = new User();
+        invalidUser = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        userIsValid.setId(77L);
+        userIsValid.setLogin("Goodvin");
+        userIsValid.setPassword("Password123");
+        userIsValid.setAge(27);
+
+        invalidUser.setId(-123L);
+        invalidUser.setLogin(null);
+        invalidUser.setPassword("1");
+        invalidUser.setAge(13);
+    }
+
+    @Test
+    void register_null_user_notOk() {
+        assertThrows(CustomException.class, () -> {
+            registrationValidator.register(null);
+        }, "User cannot be null!");
+    }
```

Comment:
In this case you expect to see you CustomException. Therefore, it should be written in the message. You message can be something similar to:
"Expected " + CustomException.class.getName() + " to be thrown for the null user, but it wasn't"
Please review your other message and correct them. Message must be more informative.